addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -89,12 +90,46 @@
                 @include media-only(print) {
                     font-weight: $font-weight-bold;
                 }
+
+                &.table-active {
+                    background: var(--ListRenderer-thead-bg-active-color);
+                    box-shadow: none;
+                }
             }
 
             .o_list_number_th {
                 /*rtl:ignore*/
                 text-align: right;
             }
+
+            .o_column_sortable {
+
+                .o_list_sortable_caret {
+                    background: var(--ListRenderer-thead-bg-color);
+                    position: absolute;
+                    right: .2rem;
+                    padding: 0 .2rem;
```

Comment:
I'm I wrong, or there is a direct relation between the `.2rem` used here and the `min-width: .4rem` below?
If so, I would suggest extracting it into a variable and the corresponding calculation.

Reply:
Not really, I just tried different sizes and chose what looked best